react-debounce-input/build.boot

```diff
@@ -0,0 +1,30 @@
+(def +lib-version+ "3.0.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies   [['cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
To ensure React is present, in a rare case where there is no other dependency on it. If you need specific version you should add exclusions or direct dependency.

https://github.com/cljsjs/packages/wiki/Foreign-library-dependencies

There are many bad packages as I don't always notice all the problems.